sci-physics/fastjet/fastjet-3.5.1.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FORTRAN_NEEDED=plugins
+PYTHON_COMPAT=( python3_{11..13} )
+DOCS_BUILDER="doxygen"
+DOCS_DEPEND="
+	media-gfx/graphviz
+	media-libs/freetype
+	virtual/latex-base
+"
+
+inherit autotools docs fortran-2 python-single-r1
+
+DESCRIPTION="A software package for jet finding in pp and e+e- collisions"
+HOMEPAGE="https://fastjet.fr/"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/fastjet/fastjet"
+else
+	SRC_URI="https://fastjet.fr/repo/${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+fi
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="cgal examples python +plugins"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+# cgal is header-only in version 5.4 and up. We need to use the
+# special --enable-cgal-header-only argument to use these versions.
+DEPEND="
+	cgal? ( >=sci-mathematics/cgal-5.4:=[shared(+)] )
+	plugins? ( sci-physics/siscone:= )
+	python? ( ${PYTHON_DEPS} )
+"
+RDEPEND="${DEPEND}"
+BDEPEND="app-shells/bash"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-3.4.0-system-siscone.patch
+	"${FILESDIR}"/${PN}-3.4.0-gfortran.patch
+)
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+	fortran-2_pkg_setup
+}
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	# only bash compatible
+	local -x "${BROOT}/bin/bash"
```

Comment:
Missing the variable name! CONFIG_SHELL=...

Reply:
ha, funny. I am running my tests right now with dash and it did not fail yet, so I guess upstream merged all bash fixes.

Reply:
Ah, no nevermind the bash problems with configure are silent...